app/services/duplicate_application.rb

```diff
@@ -7,7 +7,7 @@ def initialize(original_application_form, target_phase:, recruitment_cycle_year:
     @recruitment_cycle_year = recruitment_cycle_year
   end
 
-  IGNORED_ATTRIBUTES = %w[id created_at updated_at submitted_at course_choices_completed phase support_reference english_main_language english_language_details other_language_details].freeze
+  IGNORED_ATTRIBUTES = %w[id created_at updated_at submitted_at course_choices_completed phase support_reference english_main_language english_language_details other_language_details feedback_completed].freeze
```

Comment:
looking good! This needs to be `feedback_form_complete` though!

Reply:
good catch thanks!